editor/editor_node.cpp

```diff
@@ -1437,6 +1438,7 @@ void EditorNode::_edit_current() {
 		if (current_obj->is_class("ScriptEditorDebuggerInspectedObject")) {
 			editable_warning = TTR("This is a remote object so changes to it will not be kept.\nPlease read the documentation relevant to debugging to better understand this workflow.");
 			capitalize = false;
+			folding = true;
```

Comment:
Why enable folding for the debugger inspector? You typically want to see values in the debugger, and I think it's better to keep it using the same setting as the normal inspector.

Reply:
It's do the opposite, I'll fix this name.

Reply:
Now it's more legible I hope.